maintenance/importConfig.ts

```diff
@@ -437,59 +438,86 @@ async function parseOzwProduct(
 	for (const param of parameters) {
 		if (isNaN(param.index)) continue;
 
-		const parsedParam = ret.paramInformation[param.index] ?? {};
-
-		// don't use ?? here, some fields could be empty strings and ?? operator
-		// will not work
-		parsedParam.label = param.label || parsedParam.label;
-		parsedParam.description = param.Help || parsedParam.description;
-		parsedParam.valueSize = param.size || parsedParam.valueSize || 1;
-		parsedParam.minValue = param.min || parsedParam.min || 0;
-		parsedParam.maxValue = param.max || parsedParam.max || 100;
-		parsedParam.readOnly = Boolean(param.read_only);
-		parsedParam.writeOnly = Boolean(param.write_only);
-		parsedParam.allowManualEntry = param.type !== "list";
-		parsedParam.defaultValue = param.value || parsedParam.value || 0;
-
-		if (param.units !== undefined) {
-			parsedParam.units = param.units;
-		}
+		if (param.type === "bitset") {
+			const bitSetIds = ensureArray(param.BitSet);
+			for (const bitSet of bitSetIds) {
+				let id = parseInt(bitSet.id).toString(16);
+				id = `${param.index}[0x${padStart(id, 2, "0")}]`;
+
+				const parsedParam = ret.paramInformation[id] ?? {};
+
+				const label = ensureArray(bitSet.Label)[0];
+				const desc = ensureArray(bitSet.Help)[0];
+
+				// this values are all parsed as switches but could be transformed to
+				// list with two options: `Enable` and `Disable`
+				parsedParam.label = label;
+				parsedParam.description = desc;
+				parsedParam.valueSize = 1;
+				parsedParam.minValue = 0;
+				parsedParam.maxValue = 1;
```

Comment:
This is fine, I'm planning to automatically convert these kinds of params to boolean some time in the future.